pymc/distributions/continuous.py

```diff
@@ -2531,6 +2531,16 @@ def logcdf(value, alpha, beta):
             msg="alpha > 0, beta > 0",
         )
 
+    def icdf(value, alpha, beta):
+        res = 1 / Gamma.icdf(value=1 - value, alpha=alpha, scale=1 / beta)
```

Comment:
Better to use the stable API, and then you don't need to hack with the scale thing either

```suggestion
        res = 1 / icdf(Gamma.dist(alpha=alhpa, beta=beta), value=1 - value)
```